doc/policy/mempool-limits.md

```diff
@@ -0,0 +1,65 @@
+# Mempool Limits
+
+## Definitions
+
+Given any two transactions Tx0 and Tx1 where Tx1 spends an output of Tx0:,
```

Comment:
nit: you should decide between `:` and `,` :laughing: 
```suggestion
Given any two transactions Tx0 and Tx1 where Tx1 spends an output of Tx0:
```
or
```suggestion
Given any two transactions Tx0 and Tx1 where Tx1 spends an output of Tx0,
```

Reply:
oops yes, thank you! 😅